src/program/lwaftr/csv_stats.lua

```diff
@@ -65,17 +88,30 @@ function CSVStatsTimer:tick()
    local elapsed = engine.now() - self.start
    local dt = elapsed - self.prev_elapsed
    self.prev_elapsed = elapsed
-   self.file:write(('%f'):format(elapsed))
+   if not self.hydra_mode then
+      self.file:write(('%f'):format(elapsed))
+   end
```

Comment:
I'm quite happy to have a few flags for doing that kind of thing, particularly in tooling outside the core of the system. :-)